libs/dh/shared/assets/src/assets/b2c-custom-layouts/login-unified.html

```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
suggestion: the changes are primarily formatting (except this one) but it's probably still a good idea to upload the file after merge. What do you think @frontend-specialisten :)

Reply:
Figured that the `lang` attribute is added automatically when visiting the page, so it's really not necessary to upload the files again. So it seems that this change is mainly to please SonarCloud. But it's a good change nonetheless ;)